lib/widgets/dismissible_widget_list_item.dart

```diff
@@ -68,12 +69,15 @@ class DismissibleWidgetListItem extends StatelessWidget {
           child: Row(
             mainAxisAlignment: MainAxisAlignment.spaceBetween,
             children: [
-              Padding(
-                padding: const EdgeInsets.all(10.0),
-                child: Text(
-                  'Removed ' + widget.title,
-                  style:
-                      TextStyle(color: Theme.of(context).colorScheme.onSurface),
+              Expanded(
+                child: Padding(
+                  padding: const EdgeInsets.all(10.0),
+                  child: Text(
+                    AppLocalizations.of(context).getTranslation('dissmisibleWidgetList.removed') + widget.title,
```

Comment:
Please make this simpler:
`AppLocalizations.of(context).getTranslation('dissmisibleWidgetList.removed')`
to
`AppLocalizations.get('dissmisibleWidgetList.removed')`

Reply:
If we introduce other languages in the future we will set context once on app start - not each time when a label is needed.